src/AuthKeyValueUser.php

```diff
@@ -159,27 +157,14 @@ public function getUserRawParameters()
         return $this->userRawParameters;
     }
 
-    /**
-     * @param mixed $language
-     */
-    public function setLanguageUi($languageUri)
```

Comment:
Are we sure we are not using this method anywhere else?

Reply:
yep, it was used only in https://github.com/oat-sa/generis-auth-keyvalue/pull/31/files#diff-1627d8bb1176ffa5aee77d1d5c585c37ee07dd181b3a57447b248d5a457a1b33L95